### ISSUE TITLE
Pivot table no longer puts the factor variable in the right order

### ISSUE THREAD

**Issue by rdstern** [africanmathsinitiative/R-Instat#8420]
@MeSophie  and @N-thony Here is result from the new pivot-table dialog:

![image](https://github.com/africanmathsinitiative/R-Instat/assets/11226469/963efd71-0256-4167-85eb-3d1a23e889fe)

This is despite the dialog being like this.  so with the month added into the bottom control expressly for this purpose.  I assume it isn't running the code anymore that Lily added for this purpose.

![image](https://github.com/africanmathsinitiative/R-Instat/assets/11226469/17ce2b3b-4830-4b21-9589-f5f15bff5dc3)

I am no longer able to get the factor into the right order, i.e. Jan to Dec.  It is fine in Version 0.7.14.  Please can the feature be re-instated.  It now remains with the same new problem on the original Describe men - where it used to work fine.

**Comment by rdstern** [comment]
@MeSophie I now found (at least 2) more problems with this current dialog.  I think you made the changes for the climatic menu?

a) In the ordinary Describe menu the multiple receiver now only allows numeric variables.  It should continue to allow all variables, as it is designed for factors.
b) The Numeric Variable checkbox is also checked.  That is correct for the copy in the climatic menu, but should not be the case for the dialog, when called in the Describe menu.

This is a bit like the inability to print, that the dialog has really regressed since version 0.7.6.  It also shows I was not sufficiently alert in my testing of the changes, when the dialog was added to the climatic menu.  I assumed nothing would have changed in the main implementation of the dialog - in the Describe menu.  That assumption was totally wrong!


This needs to be fixed urgently, if possible, as we plan to use the new version of this dialog in the workshop in Mozambique!
